wiki/Errors.md

```diff
@@ -43,3 +43,13 @@ E004: Malformed manifest (invalid JSON)
 #### Causes
 
 - Provided manifest is invalid JSON (maybe was served an interstitial page, e.g. from Cloudflare)
+
+### E007
+
+```
+E007: CFG File path undefined
+```
+
+#### Causes
+
+- Path to a CFG File has not been set
```

Comment:
This error could also mean the CFG path doesn't match the regex (end with a `.cfg`)

Reply:
Sure